responsibleai_text/responsibleai_text/managers/explainer_manager.py

```diff
@@ -119,13 +119,17 @@ def compute(self):
             qa_predictor = QAPredictor(self._model)
             qa_start = qa_predictor.predict_qa_start
             qa_start.__func__.output_names = qa_predictor.output_names
-            explainer = shap.Explainer(qa_start, self._model.tokenizer)
+            explainer_start = shap.Explainer(qa_start, self._model.tokenizer)
+            qa_end = qa_predictor.predict_qa_end
+            qa_end.__func__.output_names = qa_predictor.output_names
```

Comment:
nit: I see qa_start.__func__.output_names and qa_end.__func__.output_names has same value qa_predictor.output_names can they be combined?

Reply:
done!